src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -196,7 +196,9 @@
     {{ partAdmittedClaimantAcceptsDefendantDashboard(claim) }}
   {% elif status === ClaimStatus.OFFER_SETTLEMENT_REACHED
     or status === ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_BREACHED_SETTLEMENT %}
-  {{ offerSettlementReachedDashboard() }}
+    {{ offerSettlementReachedDashboard() }}
+  {% elif status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_AS_BUSINESS_RESPONSE %}
+    {{ partAdmitRejectedByClaimantDefendantDashboard(claim) }}
```

Comment:
Please note comp/org is using same status as individual part admit rejection.

Reply:
Please update to use an or statement similar to lines 197/198 so that if someone renames the macro is only needs to be updated in one place.